bs-web-app/package.json

```diff
@@ -13,6 +13,8 @@
     "@types/node": "^12.0.0",
     "@types/react": "^17.0.0",
     "@types/react-dom": "^17.0.0",
+    "eslint-config-prettier": "^8.3.0",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Shouldn't these two also be dev deps?

Reply:
It acts up when you put it in dev for some reason. Will add it back if it becomes an issue.